src/main/java/seedu/modtrek/logic/parser/HelpCommandParser.java

```diff
@@ -49,8 +45,7 @@ public HelpCommand parse(String args) throws ParseException {
         case "SORT":
             return new HelpCommand(SortCommand.MESSAGE_USAGE);
         default:
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
+            return new HelpCommand("");
```

Comment:
Perhaps you can have another constructor for this case of empty message strings?

Reply:
What do you mean? What kind of bugs are you foreseeing?

Reply:
I think avoiding manual input for the creation of new objects is good practice? (but as far as bugs, I think eg. someone might not want to the full help command message and have the empty string as his help command input).

Reply:
It is a bit weird for someone reading the code to see the empty string input, as they might not immediately understand. @Jun-How

Reply:
I see, what if I replace it with `HelpCommand.SHOWING_ALL_MESSAGE_USAGE` instead so `selectedMessage` will be updated to it? Then `selectedMessage` will simply be returned.